app/assets/javascripts/bibliography.js

```diff
@@ -0,0 +1,45 @@
+(function (global) {
+  var Bibliography;
+
+  Bibliography = {
+    init: function (el) {
+      var $el = $(el);
+      var data = $el.data();
+      // Add a placeholder so flashes of text are not as significant
+      var placeholder = '<div class="bibliography-placeholder">' +
+                          '<h3 class="col-md-9">Bibliography</h3>' +
+                          '<div class="col-md-9 bibliography-list"></div>'+
+                        '</div>';
+      $el.html(placeholder);
+      $.getJSON(data.path, {
+          'f[related_document_id_ssim][]': data.parentid,
```

Comment:
It looks like this is the only place where we use this `parentid`.  Would it make sense for us to pass this info in via the `path` attribute and not have to construct the `facet` params here?

Reply:
I'm not sure I follow 100%. Is the idea to keep the facet params out of the JavaScript? I could construct the entire URL in the ERB view template if that's a better approach.

Reply:
I guess it just simplifies what's needed (html attribute wise) to instantiate the JS and then is/can be less tied to the implementation of "construct URL with these attributes" and could be more generically used to "replace contents of element X with response from request Y".

Totally could be me thinking about this way too abstractly though, so feel free to ignore. 😸  This is good work. 👍